build.gradle

```diff
@@ -59,6 +59,7 @@ dependencies {
 
     // libs included in the final jar
     api group: 'com.theokanning.openai-gpt3-java', name: 'service', version: '0.18.2'
```

Comment:
Can we remove the above dependency and refactor all tasks to use the official SDK instead?
It can be done later, in this case please open a followup issue.